### ISSUE TITLE
Can't Compile Tweaks with Theos on Linux, can't find working ldid

### ISSUE THREAD

**Issue by midnightchip** [theos/theos#296]
(Thanks for reporting an issue! Please make sure you click the link above to view the issue guidelines, then fill out the blanks below.)

What are the steps to reproduce this issue?
-------------------------------------------
1. …Open Theos on Linux
2. …Try to Build A tweak
3. …Profit?

What happens?
-------------
…ldid won't sign correctly, it freezes and doesnt go on.

What were you expecting to happen?
----------------------------------
…I expect ldid to sign the binaries

Any logs, error output, etc?
----------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

None
Any other comments?
-------------------
…

What versions of software are you using?
----------------------------------------
**Operating System:**Ubuntu

**Toolchain Version:** latest

**SDK Version:**9.3

**Comment by nullpixel** [comment]
this isn’t very helpful. ldid and theos works fine on Linux. run `make troubleshoot` and paste the output to ghostbin

**Comment by candoizo** [comment]
I ran into a similar sitiuation on Arch after updating theos recently, suddenly ldid was not found; naturally it was solved by compiling one and placing it in `$THEOS/bin`. without logs no one could tell if this actually targeting the problem, but rules out many potential ones

a user has added a case to facilitate building `ldid` on Linux that works for me:
```(sh)
git clone https://github.com/daeken/ldid && cd ldid
./make.sh 
mv ldid $THEOS/bin/
```

**Comment by kirb** [comment]
@candoizo, Different issue to the one filed here. Just fixed in 1c0f76a.

Closing due to lack of response and as this is probably not a Theos issue. If you need further assistance, take a look at our [help page](https://github.com/theos/theos/wiki/Help).
